modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -28,6 +28,14 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+// todo: instead of creating a texture and discarding it / instead create an image and use it directly instead of pointlessly instancing
```

Comment:
I'd suggest opening issues for each TODO instead, such comments often end up forgotten and never addressed (or addressed, but the comments themselves don't get removed).

Reply:
Fixed